entities/user.go

```diff
@@ -0,0 +1,17 @@
+package entities
+
+import (
+	"github.com/unicsmcr/hs_auth/utils"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+)
+
+// User is the struct to store registered users
+type User struct {
+	ID            primitive.ObjectID `json:"_id" bson:"_id"`
+	Name          string             `json:"name" bson:"name" validate:"required"`
+	Email         string             `json:"email" bson:"email" validate:"required,email"`
+	Password      string             `json:"password" bson:"password" validate:"required,min=6,max=50"`
```

Comment:
Minor detail, but based on [OWASP guidelines](https://github.com/OWASP/CheatSheetSeries/blob/master/cheatsheets/Password_Storage_Cheat_Sheet.md#do-not-limit-the-character-set-and-set-long-max-lengths-for-credentials) we should set the maximum length to ~160 characters